contrib/platform/src/com/sun/jna/platform/win32/User32Util.java

```diff
@@ -267,5 +307,31 @@ protected boolean getMessageFailed() {
                             lastError);
             return true;
         }
+        
+        
+        /**
+         * Set of {@link Win32VK} members that can be mapped to a UniCode code point via a keyboard layout.
+         */
+		public static final EnumSet<Win32VK> WIN32VK_MAPPABLE = EnumSet.of(Win32VK.VK_BACK, Win32VK.VK_TAB, Win32VK.VK_CLEAR, Win32VK.VK_RETURN,
```

Comment:
If I'm not mistaken the `WIN32VK_MAPPABLE` property is derived, as is the `introducedVersion` in the `Win32VK` enum. You pulled one out. I have no strong feelings either way, if you want to reintegrate the mappable property from the first revision, I would be ok with it.

Reply:
I like the mappable property like this. Very convenvient to use and clearly distinguished header values vs. "other known" values.
introducedVersion is part of the header (IFDEF), so it should be in the enum.